extensions/music/package.json

```diff
@@ -59,6 +59,12 @@
       "title": "Dislike Track",
       "description": "Dislike currently playing track.",
       "mode": "no-view"
+    },
+    {
+      "name": "add-to-library",
```

Comment:
I am wondering if you would also be open to create another command: `Love and Add to Library`

I'm not sure if it's against the guidelines to make 2 actions in a single command, but this one could be useful I think. If you want to see your loved songs they need to be in your library.

Reply:
Would be really awesome I think!

Reply:
I will open a separate PR!

Reply:
Perfect!